src/components/sections/Section.tsx

```diff
@@ -31,7 +31,7 @@ export function Section({
 
         const additionalProps: Partial<BaseSectionItemProps> = {
           radius: toRadius(index, lastIndex, firstIndex),
-          radiusSize: 'regular',
+          radiusSize: 'medium',
```

Comment:
Looks like this should be `'regular'`. Also, shouldn't this cause tsc errors? Could be related to the `Partial<BaseSectionItemProps>` type if it does elsewhere.

```suggestion
          radiusSize: 'regular',
```

Reply:
Weird that this didn't trigger a `tsc` error, it did trigger errors elsewhere. I'll make the change and investigate the `Partial<BaseSectionItemProps>` type